listenbrainz/webserver/static/js/jsx/importer.js

```diff
@@ -43,7 +43,7 @@ export default class Importer {
     this.canClose = false; // Disable the close button
     this.updateMessage("Your import from Last.fm is starting!");
     this.playCount = await this.getTotalNumberOfScrobbles();
-    this.latestImportTime = await this.APIService.getLatestImport(this.userName); // TODO: Error handling, test usernames having special characters
```

Comment:
I see no test currently for `getLatestImport`.
I think it would be good to add a test as describe in this comment before removing it.